docs/source/daml2ts/index.rst

```diff
@@ -189,23 +189,25 @@ The thing to note is how the definition of the ``Add`` case has given rise to a
 Enums
 ~~~~~
 
-DAML enumerations map naturally to TypeScript.
+Given a DAML enumeration like this,
 
 .. code-block:: daml
    :linenos:
 
    data Color = Red | Blue | Yellow
 
-The companion TypeScript type is the following.
+the generated TypeScript will consist of a type declaration and the definition of an associated companion object.
 
 .. code-block:: typescript
    :linenos:
 
-   enum Color {
-     Red = 'Red',
-     Blue = 'Blue',
-     Yellow = 'Yellow',
-   }
+   type Color = | 'Red' | 'Blue' | 'Yellow'
+
+   const Color : {readonly Red: Color; readonly Blue: Color; readonly Yellow: Color} = {
```

Comment:
I think documenting this type just adds complexity that doesn't help with anything.
```suggestion
   const Color = {
```

Reply:
https://github.com/digital-asset/daml/pull/5044